### ISSUE TITLE
NTR: multi-lobulated spleen

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4907]
**Preferred term label:**
multi-lobulated spleen

**Synonyms**
multi lobulated spleen
lobulated spleen

**Definition (free text, please give PubMed ID)**
A spleen which has a lobulated appearance with distinct lobes/lobules. PMID:25386417

**Parent term (use hpo.jax.org/app)**
HP:0025408 Abnormal spleen morphology

Note to self: For Kids First dataset: https://docs.google.com/spreadsheets/d/1Y0svlJNxiX3Yax7ZWBNdDpkkdmlHEO2EF9l6qc8JcW8/edit#gid=484265507

**Comment by pnrobinson** [comment]
added as HP_0020186
